components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+        <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+            {{ $t('modSubmissionForm.healthcareProfessionalId') }}
+        </span>
+        <div class="flex flex-col mt-4">
+            <div class="justify-start items-start flex">
+                <input 
+                    type='text'
+                    placeholder='Search by name or ID to add a doctor'
```

Comment:
Let's save the placeholder text to a variable.

Reply:
Do we want to do this for all input fields? I'm thinking about the longer forms this means we have to create 14-16 variables + the ErrorMessage variables that you mention. 

If so let me know than I will create an issue for the form so I can apply it there as well.

Reply:
yup! every error message, every piece of text.....